commands/eval.js

```diff
@@ -27,7 +27,7 @@ class Eval extends Command {
       return;
     }
     try {
-      const asyncCode = `(async() => { ${code} })()`;
+      const asyncCode = `(async() => ${code} )()`;
```

Comment:
Won't this break under certain conditions?

Reply:
It just allows for awaiting values, doesn't break anything to date in my testing.

Reply:
I'll take your word on in.